src/io/json.cpp

```diff
@@ -616,27 +590,19 @@ namespace fc
               os << "null";
               return;
          case variant::int64_type:
-         {
-              int64_t i = v.as_int64();
```

Comment:
Why change this?

Reply:
Style, mostly.
IMO these blocks are clumsy, and they're only required so you can declare the "i" variable within the case.